cme/cli.py

```diff
@@ -20,104 +15,7 @@
     datefmt='%Y-%m-%d %H:%M:%S')
```

Comment:
We can also move this into the main() method below the `args = parser.parse_args()` line to make sure it gets only called if it really is the entry point of the application.